### ISSUE TITLE
ATM page needs more semantic markup

### ISSUE THREAD

**Issue by RichCaloggero** [MIT-LCP/physionet#8]
The output here needs a bit more semantic markup:
- include role="main" or main element wrapping the entire main page content;
- introduce the main section with an h1 which wraps the relevant title of the page (ATM Output for xxx) or some such;
- There is an h1 which reads: "No annotations!" This is the only heading on the page. This should be demoted to h2, or perhaps lower if it is contained within another subsection.
- There is bold text which reads: "navigation" just before some toolbar buttons labeled like "<<", "<", "*", ">", ">>", etc. Use a heading tag instead of simply marking it as bold text.
- There is text which reads: "current observation window -- click on either side to move it". Mark this up as a heading.
- Within the observation window, somehow separate the various items either via list markup, heading markup, wrap each distinct item in a container like div with role="article", or use role="region" and then add a label to each via `aria-label`.
